internal/mock/store/store.go

```diff
@@ -0,0 +1,211 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: internal/store/store.go
+
+// Package mock_store is a generated GoMock package.
+package mock_store
```

Comment:
I'll add a `//go:generate mockgen ...` so we can generate all the mocks using `go generate ./...`

Reply:
Added following section in `internal/store/store.go` to generate mock using go generate 
```suggestion
//go:generate mockgen -destination=../mock/store/store.go -source=$GOFILE
```